### ISSUE TITLE
Getting instance index in parallel mode

### ISSUE THREAD

**Issue by k06a** [mochajs/mocha#4973]
**Is your feature request related to a problem or a nice-to-have?? Please describe.**
– I wish I had a way to get instance index in parallel mode, I will be able to use `eth-gas-reporter` in parallel mode then: https://github.com/cgewecke/hardhat-gas-reporter#parallelization

**Describe the solution you'd like**
– I want to have any way to get something like `parallelMochaIndex()` returning numbers from `0` to `N-1`.

**Describe alternatives you've considered**
– I am not sure what can be done to implement it easily.

**Comment by k06a** [comment]
Could be related: https://github.com/mochajs/mocha/pull/4813
Will try to check `process.env.MOCHA_WORKER_ID`

**Comment by JoshuaKGoldberg** [comment]
> `process.env.MOCHA_WORKER_ID`

Yes, that's what's meant to be used 🙂
